TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/DriveConstants.java

```diff
@@ -66,8 +66,8 @@
      */
     public static double MAX_VEL = 30;
     public static double MAX_ACCEL = 30;
-    public static double MAX_ANG_VEL = Math.toRadians(180);
-    public static double MAX_ANG_ACCEL = Math.toRadians(180);
+    public static double MAX_ANG_VEL = Math.toRadians(120);
+    public static double MAX_ANG_ACCEL = Math.toRadians(120);
```

Comment:
I find reducing drive constraints is often the most helpful debugging step across the board. In light of that, setting the default maxes to ~40-50% of the expected true values seems prudent.

I also have limited empirical knowledge regarding reasonable acceleration values. I guess the linear defaults seem decent and keeping the same ratio as linear makes sense.

This is somewhat off topic, but I'm reminded of an old idea to determine track width by comparing tangential/linear and angular velocities while turning. Perhaps the raw gyro outputs are of some use!

Reply:
> In light of that, setting the default maxes to ~40-50% of the expected true values seems prudent

My measured true value for a Strafer v1 was 130 deg/s for a fairly average load (30lbs). I don't think it's prudent to set the default value to a measured value for the most popular default chassis kits?

Reply:
> This is somewhat off topic, but I'm reminded of an old idea to determine track width by comparing tangential/linear and angular velocities while turning. Perhaps the raw gyro outputs are of some use!

This seems like a really neat idea!

Reply:
> My measured true value for a Strafer v1 was 130 deg/s for a fairly average load (30lbs). I think it would be reasonable to set the default value to a measured value for the most popular default chassis kits? The default 180 would already be too high for those using strafer v1.

Sorry I'm not sure I was clear enough in my initial comment. By _true_ value I mean _empirical_ value. Concretely, I'm suggesting to set the limit at ~60deg/s.

Reply:
My apologies. I totally misread what you were saying :P